index.html

```diff
@@ -50,108 +50,44 @@
+        <img src="images/corp_users.png">
     </div>
-    <div class="col-md-6">
-        <a href="https://www.usherbrooke.ca/iq/en/" target="about:blank"><img src="images/inst_quant_sher.png"></a>
-    </div>
-    <div class="col-md-6">
-        <a href="https://qfort.ncku.edu.tw/" target="about:blank"><img src="images/qfort.png" style="margin: 1rem 0;"></a>
-    </div>
-    <div class="col-md-6">
-        <a href="https://www.ncku.edu.tw/" target="about:blank"><img src="images/ncku.png" style='max-height:100px'></a>
-    </div>
-</div>
-<br>
-<p>QuTiP is currently supported by these organizations:<p></p>
-<div class="row">
-    <div class="col-md-4">
-        <a href="https://www.jst.go.jp/moonshot/en/index.html"><img src="images/moonshot-program-jst.png"></a>
-    </div>
-</div>
-
-<br>
-<div class="row">
-    <p>QuTiP is proud to be affiliated to:<p>
     <div class="col-md-5">
-        <a href="https://numfocus.org/"><img src="images/numfocus.png"></a>
-    </div>
-    <div class="col-md-5">
-        <a href="https://unitary.fund/"><img src="images/unitaryfund.png"></a>
+        <p>
+            From the classroom to the corporate office, QuTiP is used around the world to advance research in quantum optics, quantum information, quantum open systems, optomechanics, and many related fields.  QuTiP is in use at nearly every single research university around the globe, government-funded research labs, and is relied upon by every major corporation focused on developing a quantum computer.
+        </p>
     </div>
 </div>
 
-<br>
-<div class="row">
-    <p>The development of QuTiP was partially supported by the following organizations:<p>
-    <div class="col-md-4">
-        <a href="https://www.aber.ac.uk/en/about-us/faculties/business-physical-sciences/"><img src="images/aber_uni.png"></a>
-    </div>
-    <div class="col-md-3">
-        <a href="https://physics.korea.ac.kr/"><img src="images/korea-logo.png"></a>
-    </div>
-    <div class="col-md-3">
-        <a href="https://www.jsps.go.jp/english/"><img src="images/jsps.jpg"></a>
+
+
+<div class="row" style='margin-top:20px; margin-bottom: 20px;'>
+    <div class="col-md-5">
+        <p style='margin-top:20px'>
+            QuTiP is already being used at a variety of institutions around the globe, and has been downloaded more than a million times since its initial release.  Need
+            help in simulating a tricky problem? Our large community of users are gracious enough to take a moment of their time and help in tackling even the most esoteric of issues.
+        </p>
     </div>
-    <div class="col-md-4">
-        <a href="https://www.jst.go.jp/kisoken/presto/en/"><img src="images/presto.jpg"></a>
+    <div class="col-md-6">
+        <center><h4>Distribution of the 39,420 Unique Visitors in 2020</h4></center>
+        <div id='container1' style='margin-top:10px; position: relative;height: 250px;'></div>
```

Comment:
We should either remove or update this as part of the website revamp. Not for this PR necessarily.